File: src/core/jsui/ts/risks.ts
import { NumericFilterData, NullNumericFilterData } from './filters'

export interface FullRiskData {
    Risk: ProcessFlowRisk
    Flows: ProcessFlowBasicData[]
    Controls: ProcessFlowControl[]
}

export interface RiskFilterData {
    NumControls: NumericFilterData
}
export let NullRiskFilterData : RiskFilterData = {
    NumControls: JSON.parse(JSON.stringify(NullNumericFilterData))
}

File: src/core/golang/core/risks.go
package core

type Risk struct {
	Id          int64  `db:"id"`
	Name        string `db:"name"`
	Identifier  string `db:"identifier"`
	Description string `db:"description"`
	OrgId       int32  `db:"org_id"`
}

type RiskFilterData struct {
	NumControls NumericFilterData
}

var NullRiskFilterData RiskFilterData = RiskFilterData{
	NumControls: NullNumericFilterData,
}
